stonesoup/tracker/gmphd.py

```diff
@@ -0,0 +1,136 @@
+# -*- coding: utf-8 -*-
+from scipy.stats import multivariate_normal
+
+from .gaussianmixture import GaussianMixtureMultiTargetTracker
+from ..types import TaggedWeightedGaussianState
+
+
+class GMPHDTargetTracker(GaussianMixtureMultiTargetTracker):
+    """A implementation of the Gaussian Mixture
+    Probability Hypothesis Density (GM-PHD) multi-target filter
+
+    References
+    ----------
+
+    .. [1] B.-N. Vo and W.-K. Ma, “The Gaussian Mixture Probability Hypothesis
+    Density Filter,” Signal Processing,IEEE Transactions on, vol. 54, no. 11,
+    pp. 4091–4104, 2006..
+    """
+
+    def predict(self, timestamp, control_input=None, **kwargs):
```

Comment:
This could be put into a `GaussianMixturePredictor`. I wondering if this is required however, if you create a `GaussianMixtureHypothesiser` which passes the individual components to another hypothesier (which will do the prediction then for that component).